platform/android/export/gradle_export_util.h

```diff
@@ -0,0 +1,59 @@
+//
+// Created by Aman Jain on 7/6/20.
+//
+
+#ifndef GODOT_GRADLE_EXPORT_UTIL_H
+#define GODOT_GRADLE_EXPORT_UTIL_H
+
+#include "core/io/zip_io.h"
+#include "core/os/dir_access.h"
+#include "core/os/file_access.h"
+#include "core/os/os.h"
+#include "editor/editor_export.h"
+
+// Implementation of EditorExportSaveSharedObject.
+// Ignoring this setting allows all .so files to be included in gradle project
+// Used when calling the export_project_files function.
+static Error ignore_so_file(void *p_userdata, const SharedObject &p_so) {
+	return OK;
+}
+
+// Writes p_data into a file at p_path, creating directories if necessary.
+static Error store_file_at_path(const String &p_path, const Vector<uint8_t> &p_data, int compression_method = Z_DEFLATED) {
+	String dir = p_path.get_base_dir();
+	if (!DirAccess::exists(dir)) {
+		DirAccess *filesystem_da = DirAccess::create(DirAccess::ACCESS_RESOURCES);
+		filesystem_da->make_dir_recursive(dir);
```

Comment:
Should I do an error check here as well?

Reply:
Yea, good catch!

Reply:
@m4gr3d I've addressed all the feedback you've given me now.